### PR TITLE
new package: sonic-0.2.0

### PR DESCRIPTION
**Pull request by joey-commits**

### CODE REVIEW

File: srcpkgs/sonic/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	}
+}
+
+libsonic-devel_package() {
+	depends="libsonic>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+		vmove "usr/lib/*.so"
+vmove "usr/lib/*.a"
```
Comment (abenson) [inline srcpkgs/sonic/template:28]: Missing an indent here

**Comment by joey-commits** [comment]
I was wondering if there were any outstanding issues with merging this. Thank you.

**Comment by sgn** [comment]
Commit message: "New package: sonic-0.2.0"

**Comment by joey-commits** [comment]
Thank you for taking the time to provide review comments.

**Comment by joey-commits** [comment]
I updated this yesterday and was wondering if this package still had any outstanding issues before it could be merged. Thanks in advance.

**Comment by Chocimier** [comment]
Would be nice to have manpage installed (with vman).

**Comment by joey-commits** [comment]
Thanks for the feedback. I have added the man page.

**Pull request merged**